experiments/tf_trainer/common/dataset_config.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+BASE_PATH="gs://conversationai-models"
+GCS_RESOURCES="${BASE_PATH}/resources"
+MODEL_PARENT_DIR="${BASE_PATH}/tf_trainer_runs"
+
+if [ "$1" == "civil_comments" ]; then
+    train_path="${GCS_RESOURCES}/civil_comments_data/train_eval_test/train-*.tfrecord"
+    valid_path="${GCS_RESOURCES}/civil_comments_data/train_eval_test/eval-*.tfrecord"
+    labels="toxicity"
+    label_dtypes="float"
+    train_steps=24000
+    eval_period=800
+    eval_steps=50
+
+elif [ "$1" == "toxicity" ]; then
+    train_path="${GCS_RESOURCES}/toxicity_data/toxicity_q42017_train.tfrecord"
+    valid_path="${GCS_RESOURCES}/toxicity_data/toxicity_q42017_validate.tfrecord"
+    labels="frac_neg"
+    label_dtypes="float"
+    train_steps=24000
+    eval_period=800
+    eval_steps=50
+
+elif [ "$1" == "many_communities" ]; then
+    train_path="${GCS_RESOURCES}/transfer_learning_data/many_communities/20181105_train.tfrecord"
+    valid_path="${GCS_RESOURCES}/transfer_learning_data/many_communities/20181105_validate.tfrecord"
+    labels="removed"
+    label_dtypes="int"
```

Comment:
Removed is an integer? If so maybe explain what the interpretation of the integer is, e.g. 0 = not removed, 1 = removed.

Reply:
Done.